config/main.py

```diff
@@ -406,7 +408,12 @@ def add_vlan_member(ctx, vid, interface_name, untagged):
     vlan = db.get_entry('VLAN', vlan_name)
     
     if get_interface_mode() == "alias": 
-        interface_name = interface_alias(interface_name)
+        default_name = interface_alias_to_name(interface_name)
+        if default_name is None:
+            print "Invalid interface {}".format(interface_name)
```

Comment:
`interface_name` is not defined. This will crash. However, instead of replacing it with `default_name`, please refactor the code as I suggested in my comment on line 411.

Reply:
How do this crash? interface_name is the incoming argument for add_vlan_member() So, argument do have a value when invoked. 
```
def add_vlan_member(ctx, vid, interface_name, untagged):
```

Reply:
You're correct. Technically, Click should prevent `interface_name` from being `None`. I think I got a bit lost due the introduction of `default_interface`. Ignore this comment. #Closed